src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1375,6 +1377,23 @@ public function getRemovedIds(): array
 EOF;
     }
 
+    private function addDeprecatedParameters(): string
+    {
+        $deprecated = $this->container->getParameterBag()->allDeprecated();
+        if (!$deprecated) {
+            return '';
```

Comment:
should we really replace the definition of the empty array with nothing in such case ?

Reply:
Why would we compute a private property if there is nothing to export and assign?